src/integration-test/resources/GetErrandStatusesIT/__files/test1_getErrandsByErrandNumber/mappings/api-supportmanagement-get-categories-first-namespace.json

```diff
@@ -0,0 +1,13 @@
+{
+	"request": {
+		"method": "GET",
```

Comment:
Add auth-header to request-match (comment goes for all mappings)?

Reply:
Fixed